src/codeu/chat/client/simplegui/newaccountframe/NewAccount.java

```diff
@@ -0,0 +1,79 @@
+package codeu.chat.client.simplegui.newaccountframe;
+
+import java.awt.Button;
+import java.awt.GridBagConstraints;
+import java.awt.GridBagLayout;
+
+import javax.swing.JButton;
+import javax.swing.JFrame;
+import javax.swing.JLabel;
+import javax.swing.JPanel;
+import javax.swing.JTextField;
+
+public class NewAccount extends JFrame {
```

Comment:
please add class comment with regards to what this is and perhaps also how it is connected to the NewAccountPanel?